frontend/src/ManageOrgVars/ManageOrgVars.jsx

```diff
@@ -0,0 +1,409 @@
+import React from 'react';
```

Comment:
looks good if we can split the file and keep create, edit and table in different files